src/Symfony/Component/Console/Tests/Fixtures/FooVersionCommand.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class FooVersionCommand extends Command
+{
+    public $input;
+    public $output;
+
+    protected function configure()
+    {
+        $this
+            ->setName('foo:version')
+            ->setDescription('The foo:version command')
+            ->setAliases(['afooversion'])
+            ->addOption('version', null, InputOption::VALUE_OPTIONAL, 'fooopt version')
+        ;
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $this->input = $input;
+        $this->output = $output;
```

Comment:
Are those properties ever accessed from outside of this method? If not, they should be removed.

Reply:
👍 copy-pasta :)